spec.bs

```diff
@@ -517,6 +536,34 @@ to conduct an auction to select an advertisement to display to the user, they ca
 tells the browser how to conduct the auction and which on-device recorded interests are allowed to
 bid in the auction for the chance to display their advertisement.
 
+<h3 id="createauctionnonce">createAuctionNonce()</h3>
+
+{{Window/navigator}}.{{Navigator/createAuctionNonce()}} creates an auction nonce, a one-time ID
+uniquely associated with a single call to {{Window/navigator}}.{{Navigator/runAdAuction()}}. For
+multi-seller auctions, this ID is uniquely associated with all {{AuctionAdConfig/componentAuctions}}.
+This nonce will need to be passed back in via a subsequent call to
+{{Window/navigator}}.{{Navigator/runAdAuction()}} via the {{AuctionAdConfig}}. This is currently
+only needed for auctions that use additional bids, for which the auction nonce will be included in
+each additional bid as a way of ensuring that those bids are only used in the auctions for which
+they were intended.
+
+<xmp class="idl">
+[SecureContext]
+partial interface Navigator {
+  Promise<DOMString> createAuctionNonce();
+};
+</xmp>
+
+<div algorithm="createAuctionNonce()">
+The <dfn for=Navigator method>createAuctionNonce()</dfn> method steps are:
+
+  1. Let |p| be [=a new promise=].
+  1. Run the following steps [=in parallel=]:
+    1. Let |nonce| be the [=string representation=] of [=version 4 UUID=].

```

Comment:
Let's add a note below this algorithm saying that because we're going in parallel:
 1. There is no guarantee that the promise will be resolved before _other tasks_ get queued on the main thread
 2. ...which gives browsers the freedom to generate this UUID in another process, and asynchronously send it back to the main thread at an arbitrary future time